### ISSUE TITLE
demo_rasters.cpp: possible missing/moved include

### ISSUE THREAD

**Issue by deathybrs** [davidson16807/tectonics.cpp#1]
Hallo!

I am trying to understand your library better, and in the file ./demo/demo_rasters.cpp, I seem to be getting an error on line 20, which reads:

`#include <grids/SpheroidGrid/string_cast.hpp>        // to_string`

It fails to find the referenced include.  I have searched the source tree, and see several instances of a file called "string_cast.hpp."  I try changing the location from grids to rasters/entities, as that seems to be the most likely target, but then I get an error that complains that rasters::SpheroidGrid<unsigned int, float> has no member "vertex_count".

I would really love to use your API, as I have use for a tectonic sim in world generation.  Is demo_rasters.cpp the wrong example to look at?  The demo_gl.cpp file, near as I can tell, is just a minimal GL setup program, and I cannot see where it actually instantiates any of your code.  Perhaps I should be looking for string_cast.hpp somewhere else?

In any case, the code in demo_rasters.cpp appears to have what I am pretty sure is a minor issue.  Any guidance would be most appreciated.

Thanks!

**Comment by davidson16807** [comment]
Hi there, and sorry for the wait.

demo_rasters.cpp was broken for quite a while. The breakage was intentional since there were new approaches discovered for creating procedurally generated grids, which was a big win, but it required a lot of rethinking to the way we do rasters and buffering. Just yesterday I was able to implement a buffering method for dymaxion grids that's to my satisfaction. I've updated the demo/ folder so that all the preexisting examples should work. The build process has also been updated to reflect the newer way of doing things, just `cd` to the demo/grid folder and run `make`. As a reminder you will need `glew` and `glfw` installed, it's mentioned in demo/README.md.

![image](https://github.com/davidson16807/tectonics.cpp/assets/5824741/ff3000b9-013c-40ea-8d11-28164c067aad)

Change `vertices_per_square_side` in demo/grid/_test.cpp if you want to bump up the resolution:

![image](https://github.com/davidson16807/tectonics.cpp/assets/5824741/ef5052dd-c8fb-4cc2-8113-4853049f60cd)

And uncomment the different versions of `vertex_*` variables if you want to experiment with procedurally generated terrain:

![image](https://github.com/davidson16807/tectonics.cpp/assets/5824741/4dabb3b3-13e2-49f7-8385-7e8942992ed2)
